beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -250,7 +257,7 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
         // check if we have this batch
         let batch = match self.batches.get_mut(&batch_id) {
             None => {
-                debug!(self.log, "Received a block for unknown batch"; "epoch" => batch_id);
+                debug!( epoch = ?batch_id,"Received a block for unknown batch");
```

Comment:
We should use the Display impl for all these `batch_id`s in this file
```suggestion
                debug!( epoch = %batch_id,"Received a block for unknown batch");
```